docs/config-auth.md

```diff
@@ -35,3 +35,35 @@ auth:
       client_id: "12345678911-f1r0phjnhbabcdefm32etnia21keeg31.apps.googleusercontent.com"
       client_secret: "sEcRetKeYgOeShErE"
 ```
+
+## Using Gitlab as auth provider
+
+You can use Gitlab as an OIDC identity provider for BuildBuddy.
+This feature is available for both Gitlab On-Prem Deployment and Gitlab SaaS offering.
+
+For more details, please refer to [Gitlab's latest Official Documentation](https://docs.gitlab.com/ee/integration/openid_connect_provider.html)
+
+**Note**: Because [Gitlab has yet to support refresh token](https://gitlab.com/gitlab-org/gitlab/-/issues/16620), you would need to disable Refresh Token on BuildBuddy side. This will tell BuildBuddy to not request `offline_access` scope from Gitlab.
```

Comment:
```suggestion
**Note**: Because [Gitlab has yet to support refresh tokens](https://gitlab.com/gitlab-org/gitlab/-/issues/16620), you need to configure BuildBuddy to not request the `offline_access` scope from Gitlab:
```